openedx/core/djangoapps/config_model_utils/models.py

```diff
@@ -262,6 +267,25 @@ def _site_from_org(cls, org):
         else:
             return configuration.site
 
+    @classmethod
+    def _is_in_holdback(cls, user):
```

Comment:
This class is intended to be generic -- nothing specific to our features in it. Maybe move this to a simple helper function? Looks like it doesn't use `cls` at all.

Reply:
moved this method to utils.py